src/Exception/MalformedBinderException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Tsuka\DB\Exception;
+
+/**
+ * Exception to be thrown when invocating a wrongly formed Binder
+ *
+ * @package Tsuka\DB\Exception
+ */
+
+class MalformedBinderException extends \Exception
+{
+    public function __construct($message, $code = 0, \Exception $previous = null) {
```

Comment:
If you are not implementing any special logic you don't need to overwrite the constructor. Better just extend.